site/tsconfig.json

```diff
@@ -1,3 +1,8 @@
 {
-    "include" : ["./ts/**/*.ts"]
+    "include" : ["./ts/**/*.ts", "./public/js/**/*.js"],
+    "compilerOptions": {
+        "strict": true,
+        "allowJs": true,
+        "noEmit": true
+    }
 }
```

Comment:
Should add a step to the `.github` action that runs `tsc` to validate our typescript usage.

Reply:
@HenrikLam Probably best place to add it is the existing `js-lint` job

https://github.com/Submitty/Submitty/blob/2e30229699e49f376e4f7b2028b52846072f3796/.github/workflows/submitty_ci.yml#L59-L61

and just change too

```
- name: Run eslint & tsc
- run: |
    npm run eslint
    npm run build
```

Reply:
Please put these as two separate steps in the build so that you can tell at a glance what run script failed before you even expand the tab for the full details.